lib/build.js

```diff
@@ -269,16 +269,16 @@ const parseAct = config => (act_str, silent = false) => {
             .filter(l => l !== "")
             .map(l => l.trim())
         }
-        a.iff = (a.iff || []).concat( _iff )
+        a.iff = (a.iff || []).concat( _iff ).map(s => '('+s+')')
       } else if(/^if/.test(head)) {
         if(tail.join(" ").indexOf("VGas") > -1) warn(`VGas found in the conditions - gas should be handled by klab, try to remove VGas from your constraints`)
         let _if = tail
           .map(l => mapInterface(a.interface, toK(l).trim()))
           .filter(l => l != "")
-        a.if = (a.if || []).concat(_if)
-        a.if_orig = tail
-          .filter(l => l != "")
-          .map(str => str.trim())
+        a.if = (a.if || []).concat(_if).map(s => '('+s+')')
+        // a.if_orig = tail
```

Comment:
why is this excluded? its used in `klab-report`

Reply:
Oh, my bad.